[18.0][MIG] sale_exception_product_sale_manufactured_for: Migration to 18.0

LucasTran380381 opened the pull request:
depends: 
- `https://github.com/LucasTran380381/product-attribute/pull/5`
- `https://github.com/OCA/sale-workflow/pull/3415`
- `https://github.com/OCA/sale-workflow/pull/3359`

Commit:
[CHG] extract the sourcing part from stock_location_ownership in a module sale_sourced_by_line
(/tmp/trunk-generic/ rev 27)

Commit:
[FIX] the complexity 'advanced' does not exist, that's 'expert'
(/tmp/trunk-generic/ rev 29)

jgrandguillaume committed:
[FIX] Forbid to select location view in sale source by line on SO Lines
(/tmp/trunk-generic/ rev 29.1.20)

jgrandguillaume committed:
[MRG] from trunk
(/tmp/trunk-generic/ rev 40.1.6)

nbessi committed:
[FIX] set installable to True

nbessi committed:
[RM] sale_stock_ownership moved to https://launchpad.net/purchase-wkfl

Commit:
[FIX] set installable

eLBati committed:
[FIX] repository PEP8

pedrobaeza committed:
- Set as uninstallable and moved to __unported__ dir all modules.
- Update travis.yml to 8.0

yvaucher committed:
reactivate module sale_sourced_by_line

yvaucher committed:
port sale_sourced_by_line to 8.0

Now allows to set a warehouse on sale order line

Added a test for sale order with multiple warehouses sources

yvaucher committed:
move warehouse_id above route_id field in view

yvaucher committed:
improve warehouse help tooltip

yvaucher committed:
Add POT file

yvaucher committed:
bump version

yvaucher committed:
put default warehouse_id of sale order in warehouse_id of sale order line

yvaucher committed:
Give positive attitude to method is_shipped which was always False.

yvaucher committed:
Port shipped field to new API to ease testing

yvaucher committed:
Add test on _get_shipped_function

lepistone committed:
add states readonly to default warehouse

gurneyalex committed:
fix pep8 W503

gurneyalex committed:
Add OCA as author of OCA addons

In order to get visibility on https://www.odoo.com/apps the OCA board has
decided to add the OCA as author of all the addons maintained as part of the
association.

yvaucher committed:
[FIX] Do not check for procurements for sale line of type service

yvaucher committed:
Add tests for sale order with service lines

yvaucher committed:
Add module sale_procurement_group_by_line

This module extract logic to create multiple procurement group
for a single sale order for grouped sale order lines.

yvaucher committed:
Add OCA module icon

yvaucher committed:
Add bug tracker link on README.rst

sysadminmatmoz committed:
Slovene translations added

oca-transbot committed:
OCA Transbot updated translations from Transifex

sbidoul committed:
[UPD] prefix versions with 8.0

pedrobaeza committed:
[MIG] Make modules uninstallable

atchuthan committed:
[ADD] sale_exception

pedrobaeza committed:
[FIX] sale_exception: README

atchuthan committed:
[FIX] sale_exception: configuration menu

pedrobaeza committed:
[MIG] Rename manifest files

atchuthan committed:
[IMP] Rename `action_cancel` to `action_draft` to reset the flag

TheCloneMaster committed:
[FIX] sale_exception: count stock from selected warehouse

Rule should only count available stock from selected warehouse from sale order.
Without warehouse in context, it currently sums all stock available in all warehouses.

pedrobaeza committed:
sale_*: Fix parent menu

The previous parent menu has an action attached, and with these modules,
it can't be clicked.

AaronHForgeFlow committed:
[MIG] sale_sourced_by_line: Migrated to 9.0

mourad-ehm committed:
[FIX] add dependency to server-tools/base_exception

mourad-ehm committed:
[FIX] sale_exception misc

* [FIX] sale_exception confirm wizard
* [FIX] add test of test_all_draft_orders & flake8 & readme.rst
* [FIX] improve codecov test
* [FIX] delete odl translation
* [FIX] add Camptocamp to author
* [FIX] readme.rst

yvaucher committed:
[FIX] sale_exception constraint

Raise SingletonError when you have a recordset with multiple sale order.
This restore the capability to write on multiple sale.order

LoisRForgeFlow committed:
[9.0][FIX] sale_sourced_by_line: not applying default wh when grouping.

MiquelRForgeFlow committed:
[10.0][MIG] sale_sourced_by_line

simahawk committed:
[11.0][MIG] sale_exception

simahawk committed:
sale_exception: recover translations from v10

sbidoul committed:
remove obsolete .pot files [ci skip]

oca-travis committed:
[UPD] Update sale_sourced_by_line.pot

ivantodorovich committed:
[FIX] sale_exception: Menu location (#719)

Currently, it's located at the root menu because. It's a migration bug because the old menu was removed from 11.0.

ljimenezsidoo committed:
Sale Exception to v12

sebalix committed:
[MIG] sale_exception: Migration to 12.0

sebalix committed:
[IMP] sale_exception: format XML file 'views/sale_view.xml'

florian-dacosta committed:
Adapt sale_exception to base_exception simplification

florian-dacosta committed:
Add test and fix apple and orange mixing error between sale order and sale order line

Commit:
sale_exception: Remove side effect from api.constrains

The method called by 'sale_check_exception' has a side effect, it writes
on 'exception.rule' + on the Many2many relation between it and
sale.order(.line). When decorated by @api.constrains, any error during
the method will be caught and re-raised as "ValidationError".
This part of code is very prone to concurrent updates as 2 sales having
the same exception will both write on the same 'exception.rule'.
A concurrent update (OperationalError) is re-raised as ValidationError,
and then is not retried properly.

Calling the same method in create/write has the same effect than
@api.constrains without shadowing the exception type.

Full explanation:
https://github.com/OCA/server-tools/issues/1642

zeroheure committed:
[FIX] sale_exception: remove obsolete base_sale_multi_channel reference (#963)

ivantodorovich committed:
[IMP] sale exception (#878)

* [IMP] sale_exception: show menu only to Exception Rule Managers
* [IMP] Reindent lines to 4 spaces
* [IMP] Improve information messages that display exceptions
* [REM] Remove exception_ids fields from the sale order view, because it's no longer necessary and it allowed users to accidentaly deactivate a Sale Exception Rule using the active widget.
* [IMP] Rename exception_ids_formatted to exceptions_summary

pedrobaeza committed:
[FIX] sale_exception: Disable failing test

Disable for now as the solution is not obvious.

Partially fixes #972

pedrobaeza committed:
[FIX] sale_exception: Avoid W0101 pylint error

pedrobaeza committed:
[FIX] sale_exception: Disable test via __init__

Other ways it provokes linter fail

saxomanu committed:
[12.0] [MIG] sale_sourced_by_line

jaredkipe committed:
[MIG] sale_exception: Migration to 13.0

Commit:
sale_exception: keep detail of exceptions on lines

So we can:

* display the lines with an exception in red
* show the exception messages when we click on a line

It's a huge usability improvement when users have dozens or hundreds of
lines.

Commit:
sale_exception: apply pre-commit

SButko committed:
[REF] sale_automatic_workflow|_stock: separate

There is an issue for service companies that do not need the stock
module to be installed when using `sale_automatic_workflow`. Module
depended on `stock`, therefore it was auto installed. Now, stock related
part is moved to a separate module - `sale_automatic_workflow_stock` and
can be installed separately.

simahawk committed:
sale_exception: fix test class

TestSaleOrder was useless and making sale order tests run twice.

simahawk committed:
sale_exception: drop backward compat code

simahawk committed:
sale_exception: fix warning

oca-travis committed:
[UPD] Update sale_exception.pot

OCA-git-bot committed:
[UPD] README.rst

simahawk committed:
sale_exception: restore multi record tests, fix base class

oca-travis committed:
[UPD] Update sale_exception.pot

OCA-git-bot committed:
sale_exception 13.0.1.1.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-13.0/sale-workflow-13.0-sale_exception
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-13-0/sale-workflow-13-0-sale_exception/

sebastienbeau committed:
[MIG] sale_exception: Migration to 14.0

danimv5 committed:
Added translation using Weblate (Catalan)

Translated using Weblate (Catalan)

Currently translated at 100.0% (5 of 5 strings)

Translation: sale-workflow-14.0/sale-workflow-14.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_commercial_partner/ca/

Yvesldff committed:
Added translation using Weblate (French (France))

Translated using Weblate (French (France))

Currently translated at 40.0% (2 of 5 strings)

Translation: sale-workflow-14.0/sale-workflow-14.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_commercial_partner/fr_FR/

Translated using Weblate (French)

Currently translated at 100.0% (2 of 2 strings)

Translation: sale-workflow-14.0/sale-workflow-14.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_commercial_partner/fr/

TDu committed:
[13.0][FIX] sale_exception ignore after confirm

After the sale order has been confirmed, some fields can still be
changed, those changes can raise some more excetptions again.
Unfortunatelly those changes are raised as ValidationError and they are
blocking.

If during the confirmation of the sale.order the ignore excetpiton has
been used. Those same ignored exception will block some changes made
when the sale order is in `sale` state.

And this problem is due to that onchange that this PR removes.
Why should the ignore_exception be set to false when lines are changed
but not when some fields on the sale.order record are changed ?

damdam-s committed:
fix sale_exception datas

oca-travis committed:
[UPD] Update sale_exception.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
sale_exception 14.0.1.0.1

ValentinVinagre committed:
Translated using Weblate (Spanish)

Currently translated at 21.0% (8 of 38 strings)

Translation: sale-workflow-14.0/sale-workflow-14.0-sale_exception
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_exception/es/

murtuzasaleh committed:
[FIX] Travis

murtuzasaleh committed:
[14.0][MIG] sale_sourced_by_line

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (2 of 2 strings)

Translation: sale-workflow-14.0/sale-workflow-14.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_commercial_partner/de/

marylla committed:
Translated using Weblate (German)

Currently translated at 73.6% (28 of 38 strings)

Translation: sale-workflow-14.0/sale-workflow-14.0-sale_exception
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_exception/de/

victoralmau committed:
[IMP] sale_exception: Add Customer sale warning + product warning exceptions

santostelmo committed:
[MIG] sale_commercial_partner: Migration to 15.0

Commit:
[ADD] sale_exception_product_sale_manufactured_for

Commit:
[14.0] [MIG] sale_exception_product_sale_manufactured_for: Migrate to version 14.0

victoralmau committed:
[ADD] sale_partner_selectable_option: This module limits the partners to be selected in the sales orders.
TT32528

oca-travis committed:
[UPD] Update sale_exception.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
sale_exception 14.0.1.1.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-14.0/sale-workflow-14.0-sale_exception
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_exception/

victoralmau committed:
[IMP] sale_partner_selectable_option: Change position field.
TT33223

victoralmau committed:
[OU-FIX] sale_partner_selectable_option: Change 13.0.2 migration script.

victoralmau committed:
[FIX] sale_partner_selectable_option: Improve domain in partner_id field from sale order.
TT33223

victoralmau committed:
[IMP] sale_partner_selectable_option: Improve sale button in contact form to show only if sale_selectable is checked.
TT33933

JoanMForgeFlow committed:
[MIG][15.0] sale_source_by_line: migrate to version 15.0

damdam-s committed:
[MIG] sale_exception: Migration to 15.0

Commit:
[UPD] Update sale_sourced_by_line.pot

OCA-git-bot committed:
sale_sourced_by_line 15.0.1.0.1

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-15.0/sale-workflow-15.0-sale_sourced_by_line
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-15-0/sale-workflow-15-0-sale_sourced_by_line/

cesar-tecnativa committed:
[MIG] sale_partner_selectable_option: Migration to 15.0

Commit:
[UPD] Update sale_exception.pot

OCA-git-bot committed:
[UPD] README.rst

Commit:
[FIX] Update deprecated unit tests

OCA-git-bot committed:
sale_partner_selectable_option 15.0.1.0.1

alan196 committed:
Added translation using Weblate (Spanish (Mexico))

Translated using Weblate (Spanish (Mexico))

Currently translated at 100.0% (2 of 2 strings)

Translation: sale-workflow-15.0/sale-workflow-15.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-15-0/sale-workflow-15-0-sale_commercial_partner/es_MX/

matiasperalta1 committed:
[MIG] sale_exception: Migration to 16.0

MiquelRForgeFlow committed:
[FIX] sale_sourced_by_line: adapt method to v15

OCA-git-bot committed:
sale_sourced_by_line 15.0.1.1.0

alexis-via committed:
[MIG] sale_commercial_partner to v16

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (2 of 2 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_commercial_partner/it/

Translated using Weblate (Italian)

Currently translated at 100.0% (1 of 1 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_commercial_partner/it/

Translated using Weblate (Italian)

Currently translated at 100.0% (2 of 2 strings)

Translation: sale-workflow-17.0/sale-workflow-17.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-17-0/sale-workflow-17-0-sale_commercial_partner/it/

badbole committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (2 of 2 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_commercial_partner/hr/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 50.0% (2 of 4 strings)

Translation: sale-workflow-15.0/sale-workflow-15.0-sale_sourced_by_line
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-15-0/sale-workflow-15-0-sale_sourced_by_line/it/

sbejaoui committed:
[16.0][MIG] - finalize sale_exception migration

sbejaoui committed:
[FIX] - sale_exception: fix unit tests

Commit:
[UPD] Update sale_exception.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
sale_exception 16.0.1.0.1

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_exception
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_exception/

badbole committed:
Translated using Weblate (Croatian)

Currently translated at 46.5% (20 of 43 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_exception
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_exception/hr/

aliciagaarzo committed:
[16.0][IMP] sale_exception: Improved wizard view and don't ignore exceptions with "is blocking" checked

Commit:
Translated using Weblate (Spanish)

Currently translated at 73.8% (31 of 42 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_exception
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_exception/es/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 16.6% (7 of 42 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_exception
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_exception/it/

sysadminmatmoz committed:
Translated using Weblate (Slovenian)

Currently translated at 16.6% (7 of 42 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_exception
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_exception/sl/

rousseldenis committed:
Translated using Weblate (French)

Currently translated at 78.5% (33 of 42 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_exception
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_exception/fr/

glitchov committed:
Translated using Weblate (French)

Currently translated at 78.5% (33 of 42 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_exception
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_exception/fr/

Commit:
[16.0][IMP]sale_cancel_reason: set reason to false when status is != cancel

OCA-git-bot committed:
sale_cancel_reason 16.0.1.0.1

Yadier-Tecnativa committed:
[MIG] sale_partner_selectable_option: Migration to 16.0

glitchov committed:
[FIX] sale_exception : so line decoration-danger condition

OCA-git-bot committed:
sale_exception 16.0.1.1.0

Commit:
[UPD] Update sale_partner_selectable_option.pot

OCA-git-bot committed:
[UPD] README.rst

Commit:
[UPD] Update sale_exception.pot

OCA-git-bot committed:
sale_exception 16.0.1.2.0

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_exception
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_exception/

glitchov committed:
Translated using Weblate (French)

Currently translated at 83.7% (36 of 43 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_exception
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_exception/fr/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 50.0% (2 of 4 strings)

Translation: sale-workflow-15.0/sale-workflow-15.0-sale_sourced_by_line
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-15-0/sale-workflow-15-0-sale_sourced_by_line/it/

Fabiola-Auz committed:
[MIG] sale_sourced_by_line: Migration to 16.0

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (18 of 18 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_cancel_reason
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_cancel_reason/pt_BR/

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 50.0% (1 of 2 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_commercial_partner/pt_BR/

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (2 of 2 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_commercial_partner/pt_BR/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (18 of 18 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_cancel_reason
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_cancel_reason/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (2 of 2 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_commercial_partner/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (43 of 43 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_exception
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_exception/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (4 of 4 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_partner_selectable_option
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_partner_selectable_option/es/

simahawk committed:
sale_exception_product_sale_manufactured_for: adapt to variant

Partner relation has been moved to variant here https://github.com/OCA/product-attribute/pull/1400

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

pilarvargas-tecnativa committed:
[FIX] sale_commercial_partner: add development_status

TT44347

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_cancel_reason
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_cancel_reason/

Commit:
Translated using Weblate (French)

Currently translated at 100.0% (2 of 2 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_commercial_partner
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_commercial_partner/fr/

kikopeiro committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (43 of 43 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_exception
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_exception/es/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (4 of 4 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_partner_selectable_option
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_partner_selectable_option/it/

Commit:
[UPD] Update sale_sourced_by_line.pot

jappi00 committed:
Translated using Weblate (German)

Currently translated at 100.0% (4 of 4 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_sourced_by_line
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_sourced_by_line/de/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (4 of 4 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_sourced_by_line
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_sourced_by_line/it/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (4 of 4 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_sourced_by_line
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_sourced_by_line/es/

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (4 of 4 strings)

Translation: sale-workflow-16.0/sale-workflow-16.0-sale_sourced_by_line
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_sourced_by_line/pt_BR/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (6 of 6 strings)

Translation: sale-workflow-14.0/sale-workflow-14.0-sale_exception_product_sale_manufactured_for
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_exception_product_sale_manufactured_for/es/

celm1990 committed:
[IMP] sale_sourced_by_line: pre-commit auto fixes

celm1990 committed:
[MIG] sale_sourced_by_line: Migration to 17.0

AungKoKoLin1997 committed:
[IMP] sale_partner_selectable_option

nguyenminhchien committed:
[MIG] sale_exception: Migration to 17.0

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[IMP] sale_commercial_partner: pre-commit auto fixes

Commit:
[MIG] sale_commercial_partner: Migration to 17.0

maciej-wichowski committed:
[IMP] sale_cancel_reason: pre-commit auto fixes

maciej-wichowski committed:
[MIG] sale_cancel_reason: migration to 17.0

nguyenminhchien committed:
[IMP] sale_exception: pre-commit auto fixes

Commit:
[UPD] Update sale_cancel_reason.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-17.0/sale-workflow-17.0-sale_cancel_reason
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-17-0/sale-workflow-17-0-sale_cancel_reason/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-17.0/sale-workflow-17.0-sale_cancel_reason
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-17-0/sale-workflow-17-0-sale_cancel_reason/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (18 of 18 strings)

Translation: sale-workflow-17.0/sale-workflow-17.0-sale_cancel_reason
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-17-0/sale-workflow-17-0-sale_cancel_reason/it/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (18 of 18 strings)

Translation: sale-workflow-17.0/sale-workflow-17.0-sale_cancel_reason
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-17-0/sale-workflow-17-0-sale_cancel_reason/es/

Commit:
[UPD] Update sale_exception.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (43 of 43 strings)

Translation: sale-workflow-17.0/sale-workflow-17.0-sale_exception
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-17-0/sale-workflow-17-0-sale_exception/it/

Deriman-Alonso committed:
[IMP] sale_partner_selectable_option: pre-commit auto fixes

Deriman-Alonso committed:
[MIG] sale_partner_selectable_option: Migration to 17.0

jakobkrabbe committed:
Translated using Weblate (Swedish)

Currently translated at 100.0% (18 of 18 strings)

Translation: sale-workflow-17.0/sale-workflow-17.0-sale_cancel_reason
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-17-0/sale-workflow-17-0-sale_cancel_reason/sv/

jakobkrabbe committed:
Translated using Weblate (Swedish)

Currently translated at 100.0% (43 of 43 strings)

Translation: sale-workflow-17.0/sale-workflow-17.0-sale_exception
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-17-0/sale-workflow-17-0-sale_exception/sv/

Commit:
[UPD] Update sale_automatic_workflow_stock.pot

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (13 of 13 strings)

Translation: sale-workflow-17.0/sale-workflow-17.0-sale_automatic_workflow_stock
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-17-0/sale-workflow-17-0-sale_automatic_workflow_stock/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (43 of 43 strings)

Translation: sale-workflow-17.0/sale-workflow-17.0-sale_exception
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-17-0/sale-workflow-17-0-sale_exception/it/

Commit:
[UPD] Update sale_sourced_by_line.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-17.0/sale-workflow-17.0-sale_sourced_by_line
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-17-0/sale-workflow-17-0-sale_sourced_by_line/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (6 of 6 strings)

Translation: sale-workflow-17.0/sale-workflow-17.0-sale_sourced_by_line
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-17-0/sale-workflow-17-0-sale_sourced_by_line/it/

matiasperalta1 committed:
[MIG] sale_exception: Migration to 18.0

BertVGroenendael committed:
[IMP] sale_commercial_partner: pre-commit auto fixes

Co-authored-by: Lukas Tran <nhant@trobz.com>

BertVGroenendael committed:
[MIG] sale_commercial_partner: Migration to 18.0

Co-authored-by: Lukas Tran <nhant@trobz.com>

chaule97 committed:
[MIG] sale_automatic_workflow: Migration to 18.0

chaule97 committed:
[IMP] sale_automatic_workflow_stock: pre-commit auto fixes

chaule97 committed:
[MIG] sale_automatic_workflow_stock: Migration to 18.0

Commit:
[MIG] sale_exception_product_sale_manufactured_for: Migration to 18.0

Commit:
[IMP] sale_exception_product_sale_manufactured_for: pre-commit execution

xaviedoanhduy commented on sale_exception_product_sale_manufactured_for/tests/test_sale_exception.py:
```diff
@@ -1,10 +1,10 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
 
-from odoo.tests import SavepointCase
+from odoo.tests import TransactionCase
```
FYI, to avoid the `tracking_disable` context, you can use `BaseCommon` class 
`from odoo.addons.base.tests.common import BaseCommon`

LucasTran380381 replied:
thanks for information

xaviedoanhduy commented on sale_exception_product_sale_manufactured_for/models/sale_order.py:
```diff
@@ -0,0 +1,46 @@
+
+        self.env.cr.execute(
+            """
+            SELECT rel.product_id, ARRAY_AGG(rel.partner_id)
```
Is it possible to use `SQL` class to wrap query?

LucasTran380381 replied:
will fix

xaviedoanhduy reviewed (approved):
LGTM

Commit:
[UPD] Update sale_partner_selectable_option.pot

OCA-git-bot committed:
[BOT] post-merge updates

Borruso committed:
[MIG] sale_order_line_menu: Migration to 18.0

ajay-odoogap committed:
[MIG] sale_sourced_by_line: Migration to 18.0

[FIX] pylint error

bizzappdev committed:
[IMP] sale_cancel_reason: pre-commit auto fixes

bizzappdev committed:
[MIG] sale_cancel_reason: Migration to 18.0

OCA-git-bot committed:
Merge PR #3504 into 18.0

Signed-off-by alexis-via

Commit:
[UPD] Update sale_commercial_partner.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #3420 into 18.0

Signed-off-by rvalyi

Commit:
[UPD] Update sale_procurement_group_by_line.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-18.0/sale-workflow-18.0-sale_procurement_group_by_line
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-18-0/sale-workflow-18-0-sale_procurement_group_by_line/

OCA-git-bot committed:
Merge PR #3441 into 18.0

Signed-off-by rvalyi

Commit:
[UPD] Update sale_automatic_workflow.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-18.0/sale-workflow-18.0-sale_automatic_workflow
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-18-0/sale-workflow-18-0-sale_automatic_workflow/

OCA-git-bot committed:
Merge PR #3359 into 18.0

Signed-off-by StefanRijnhart

Commit:
[UPD] Update sale_exception.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-18.0/sale-workflow-18.0-sale_exception
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-18-0/sale-workflow-18-0-sale_exception/

OCA-git-bot committed:
Merge PR #3447 into 18.0

Signed-off-by simahawk

Commit:
[UPD] Update sale_automatic_workflow_stock.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #3506 into 18.0

Signed-off-by simahawk

Commit:
[UPD] Update sale_cancel_reason.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-18.0/sale-workflow-18.0-sale_cancel_reason
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-18-0/sale-workflow-18-0-sale_cancel_reason/

OCA-git-bot committed:
Merge PR #3470 into 18.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update sale_order_line_menu.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-18.0/sale-workflow-18.0-sale_order_line_menu
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-18-0/sale-workflow-18-0-sale_order_line_menu/

OCA-git-bot committed:
Merge PR #3496 into 18.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update sale_sourced_by_line.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: sale-workflow-18.0/sale-workflow-18.0-sale_sourced_by_line
Translate-URL: https://translation.odoo-community.org/projects/sale-workflow-18-0/sale-workflow-18-0-sale_sourced_by_line/

OCA-git-bot committed:
Merge PR #3464 into 18.0

Signed-off-by StefanRijnhart

Commit:
[UPD] Update sale_order_line_date.pot

OCA-git-bot committed:
[BOT] post-merge updates

juancarlosonate-tecnativa committed:
[IMP] sale_partner_selectable_option: pre-commit auto fixes

juancarlosonate-tecnativa committed:
[MIG] sale_partner_selectable_option: Migration to 18.0

TT54294

OCA-git-bot committed:
Merge PR #3575 into 18.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update sale_partner_selectable_option.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[IMP] update dotfiles
